charts/datadog/values.yaml

```diff
@@ -396,6 +396,20 @@ datadog:
     enabled: false
     # datadog.prometheusScrape.serviceEndpoints -- Enable generating dedicated checks for service endpoints.
     serviceEndpoints: false
+    # datadog.prometheusScrape.additionalConfigs -- Allows adding advanced openmetrics check configurations with custom discovery rules.
```

Comment:
could you add in the description which is the min required agent version?

Reply:
good point, done!